inst/builtin/bs5/shiny/_rules.scss

```
@@ -189,3 +189,69 @@ $icon-classes: ("bi", "fa", "fas", "far", "fab", "material-icons") !default;
     --bslib-color-fg: #008558; // Contrast 4.67
   }
 }
+
+// Modals
+.modal-body, .modal-footer {
+  padding: 1.5rem;
+}
+.modal-header .btn-close {
+  align-self: start;
+  display: flex;
+}
+#shiny-modal-wrapper:has( ~ .modal-backdrop) .modal {
+  // Blur the background when the modal has a backdrop
+  backdrop-filter: blur(2px);
+}
+
+// Progress Bars & Notifications
+// FIXME: Use css layers (dynamic themeing issue)
+#shiny-notification-panel#shiny-notification-panel {
+  position: fixed;
+  bottom: calc(var(--bslib-spacer, 1rem) / 2);
+  right: calc(var(--bslib-spacer, 1rem) / 2);
+  width: 450px;
+  z-index: $zindex-toast;
+}
+
+.progress-message {
+  margin-right: .6rem;
+}
+
+// FIXME: Use css layers (dynamic themeing issue)
+.shiny-notification.shiny-notification {
+  position: relative;
+  opacity: 0.96;
+  padding: calc(var(--bslib-spacer, 1rem) * 2);
+  margin: var(--bslib-spacer, 1rem);
+  border: 1px solid rgba(var(--#{prefix}emphasis-color-rgb), 0.8);
+  border-radius: 8px;
```

This is to match this variable in the preset, right?

```suggestion
  border-radius: $card-border-radius;
```

---

Good catch, thanks! (This PR predates setting that variable in the preset.)